dts/bindings/memory-controllers/st,stm32-fmc.yaml

```diff
@@ -46,3 +46,20 @@ properties:
 
     pinctrl-names:
       required: true
+
+    st,mem-swap:
+      type: string
+      required: false
+      default: "disable" # reset state
+      enum:
+        - "disable"
+        - "sdram-sram"
+        - "sdramb2"
+      description: |
+        The FMC bank mapping configuration (BMAP bits of FMC_BCR1).
+
+        * disable    - Default mapping.
+        * sdram-sram - Swap the NOR/PSRAM bank with SDRAM.
+        * sdramb2    - Remaps the SDRAM bank2.
+
+        Note: Supported only by H7 series.
```

Comment:
If this is the case, maybe a dedicated `st-stm32h7-fmc` dedicated binding could be added (that would include current binding and add this property).

Reply:
Done